.github/workflows/release.yml

```diff
@@ -0,0 +1,53 @@
+
+name: release
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup JFrog CLI
+        uses: jfrog/setup-jfrog-cli@v1
+      - name: Get the version
+        id: get_version
+        run: |
+          IFS=. read major minor build <<< "${{ github.event.release.tag_name }}"
+          echo ::set-output name=build::"${build}"
+      - name: Promote Artifact
+        id: jfrog
+        env:
+          ARTIFACTORY_API_KEY: ${{ secrets.ARTIFACTORY_API_KEY }}
+        run: |
+          jfrog rt bpr --apikey $ARTIFACTORY_API_KEY --url https://repox.jfrog.io/artifactory --status release parent ${{ steps.get_version.outputs.build }} sonarsource-private-releases 
```

Comment:
Is the repo `sonarsource-private-releases` correct? This is a public artifact.

Reply:
To keep it simple, you can use the same command as in line 34 to authenticate to repox and remove the url and apikey parameters here.